source/features/quick-reactions.tsx

```diff
@@ -0,0 +1,98 @@
+import delegate, {DelegateEvent} from 'delegate-it';
+import React from 'dom-chef';
+import * as pageDetect from 'github-url-detection';
+
+import features from '../feature-manager.js';
+
+const emojiSortOrder = ['THUMBS_UP', 'THUMBS_DOWN', 'LAUGH', 'HOORAY', 'CONFUSED', 'HEART', 'ROCKET', 'EYES'];
+
+function optimisticReactionFromPost(event: DelegateEvent<MouseEvent>): void {
+	const reaction = event.delegateTarget as HTMLButtonElement;
```

Comment:
IIRC you can just pass the type here, no need to assert it. Refer to its docs if it fails, also fix it elsewhere

```suggestion
function optimisticReactionFromPost(event: DelegateEvent<MouseEvent, HTMLButtonElement>): void {
```